### ISSUE TITLE
Recover and finish the work done on PR #25

### ISSUE THREAD

**Issue by ar9708** [OwnTube-tv/web-client#47]
Please @mykhailodanilenko, can you recover the "lost PR" #25 and complete it? The student Viktor who were working on it is no longer with us.

**Comment by mykhailodanilenko** [comment]
@ar9708 I think the more recent developments on this issue are in Viktor's other PR https://github.com/OwnTube-tv/web-client/pull/46, I should work with that instead (the branch for #25 is completely deleted btw)

**Comment by ar9708** [comment]
> @ar9708 I think the more recent developments on this issue are in Viktor's other PR #46, I should work with that instead (the branch for #25 is completely deleted btw)

Yeah, I see what you mean, it was probably not the best idea to delete the entire repo. However, as you indicate, from looking at the GitHub PR views it looks like #46 have all the important graphics stuff to recover the essentials:

<img width="1981" alt="image" src="https://github.com/OwnTube-tv/web-client/assets/123388211/1e6dd743-cc1a-4436-9f63-c251d26743e8">
